<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>setup_driverClass</class>
 <widget class="QDialog" name="setup_driverClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>636</width>
    <height>221</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>setup_driver</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="2" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="5" colspan="2">
    <widget class="QSpinBox" name="spinBox_DT">
     <property name="minimum">
      <number>20</number>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
     <property name="value">
      <number>150</number>
     </property>
    </widget>
   </item>
   <item row="4" column="7">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>ms</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="testButton">
     <property name="text">
      <string>Test Btn</string>
     </property>
    </widget>
   </item>
   <item row="5" column="4" colspan="2">
    <widget class="QPushButton" name="okButton">
     <property name="text">
      <string>OK</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="6" colspan="2">
    <widget class="QPushButton" name="cancelButton">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" rowspan="3">
    <layout class="QGridLayout" name="gridLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="1" column="0">
      <widget class="QPushButton" name="RA_DEC_Button">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>RA-</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="RA_INC_Button">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>RA+</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="DEC_INC_Button">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>DEC+</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="DEC_DEC_Button">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>DEC-</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>Device</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QComboBox" name="comboBox_DeviceList"/>
   </item>
   <item row="1" column="4" colspan="4">
    <widget class="QLineEdit" name="lineEdit_IoDevice">
     <property name="maxLength">
      <number>63</number>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="l_Info">
     <property name="text">
      <string>Info</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3" rowspan="2" colspan="5">
    <widget class="QLabel" name="label_Information">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QCheckBox" name="inverseBits">
     <property name="text">
      <string>inverse bits</string>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <widget class="QCheckBox" name="checkBox_UseDT">
     <property name="text">
      <string>use DT</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="8">
    <widget class="QGroupBox" name="bitGroupBox">
     <property name="title">
      <string>Bit mapping</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="margin">
       <number>4</number>
      </property>
      <property name="spacing">
       <number>4</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>bit 0</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>bit 1</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>bit 2</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>bit 3</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>bit 4</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>bit 5</string>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>bit 6</string>
        </property>
       </widget>
      </item>
      <item row="0" column="7">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>bit 7</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QComboBox" name="bitBox0"/>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="bitBox1"/>
      </item>
      <item row="1" column="2">
       <widget class="QComboBox" name="bitBox2"/>
      </item>
      <item row="1" column="3">
       <widget class="QComboBox" name="bitBox3"/>
      </item>
      <item row="1" column="4">
       <widget class="QComboBox" name="bitBox4"/>
      </item>
      <item row="1" column="5">
       <widget class="QComboBox" name="bitBox5"/>
      </item>
      <item row="1" column="6">
       <widget class="QComboBox" name="bitBox6"/>
      </item>
      <item row="1" column="7">
       <widget class="QComboBox" name="bitBox7"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>bitBox0</tabstop>
  <tabstop>bitBox1</tabstop>
  <tabstop>bitBox2</tabstop>
  <tabstop>bitBox3</tabstop>
  <tabstop>bitBox4</tabstop>
  <tabstop>bitBox5</tabstop>
  <tabstop>bitBox6</tabstop>
  <tabstop>bitBox7</tabstop>
  <tabstop>comboBox_DeviceList</tabstop>
  <tabstop>lineEdit_IoDevice</tabstop>
  <tabstop>inverseBits</tabstop>
  <tabstop>checkBox_UseDT</tabstop>
  <tabstop>spinBox_DT</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
  <tabstop>testButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
